dist/pythonlibs/testrunner/spawn.py

```diff
@@ -56,8 +56,8 @@ def _reset_board(env):
         pass
 
 
-def list_until(l, cond):
-    return l[:([i for i, e in enumerate(l) if cond(e)][0])]
+def list_until(trace_list, cond):
+    return trace_list[:([i for i, e in enumerate(trace_list) if cond(e)][0])]
```

Comment:
IIRC this can be used generally, not just for trace lists, so just

```suggestion
def list_until(list_, cond):
    return list_[:([i for i, e in enumerate(list_) if cond(e)][0])]
```

Or `lst` or something like that?

Reply:
Seems like [your suggestion is PEP8 valid](https://www.python.org/dev/peps/pep-0008/#id46), so let's do that.

Reply:
Addressed.

Reply:
Yepp, when you read a lot of foreign python code, you see this pattern of appending a `_` to a keyword to make it a too good variable name quite often ;-).

Reply:
Though come to think of it: list should probably be fine as well, as the casting function `list` is not a keyword, but some linters might complain ;-).